tst/testinstall/varargs.tst

```diff
@@ -0,0 +1,54 @@
+gap> START_TEST("varargs.tst");
+gap> f := function(a,b...) return[a,b]; end;
+function( a, b... ) ... end
+gap> Print(f,"\n");
+function ( a, b... )
+    return [ a, b ];
+end
+gap> f(1);
+[ 1, [  ] ]
+gap> f(1,2);
+[ 1, [ 2 ] ]
+gap> f(1,2,3);
+[ 1, [ 2, 3 ] ]
+gap> f := function(arg) return arg; end;
+function( arg... ) ... end
+gap> Print(f,"\n");
+function ( arg... )
+    return arg;
+end
+gap> f();
+[  ]
+gap> f(1);
+[ 1 ]
+gap> f(1,2);
+[ 1, 2 ]
+gap> f := function(a...) return a; end;
+function( a... ) ... end
+gap> f(1);
+[ 1 ]
+gap> f(1,2,3);
+[ 1, 2, 3 ]
+gap> Print(f,"\n");
+function ( a... )
+    return a;
+end
```

Comment:
Again pedantic, but: why are the tests for `function(arg) return arg; end;` and `f := function(a...) return a; end;` doing different things, in a different order? Ideally, the two functions should be equivalent (except for the output of `Print`), so I'd expect them to have equivalent / equal tests.